lib/scss/6_components/_conversations.scss

```
@@ -1498,3 +1482,36 @@ $color-background-button-modalities: rgba(255, 255, 255, 0.3);
     font-weight: bold;
     font-size: $spacing-unit / 2;
 }
+
+.Progress-status{
+    display: table;
+    margin: 0 auto;
+
+    .Progress-porcentage{
+        font-weight: bold;
+        display: table-cell;
+        font-size: $spacing-unit-tiny * 2;
+        padding-right: $spacing-unit-tiny;
+    }
+    .Progress-absolute{
+        display: table-cell;
+        font-size: $spacing-unit-tiny * 2;
```

Para os font-sizes, temos valores definidos nas variáveis no arquivo de typography...
Se não me engano, para esse tamanho definido aqui, seria o
```suggestion
        font-size: $font-size-default;
```